app/models/local_transaction_service.rb

```diff
@@ -1,41 +1,43 @@
 class LocalTransactionService
-  def self.config
-    @config ||= begin
-      filepath = Rails.root.join(Frontend.local_transactions_config)
-      YAML.safe_load(File.open(filepath))["services"]
-    end
-  end
+  attr_reader :lgsl, :devolved_administration_availability, :country_name
 
-  attr_reader :lgsl
-
-  def initialize(name, lgsl, country_name, local_authority_homepage)
+  def initialize(name, lgsl, country_name, local_authority_homepage, devolved_administration_availability)
     @name = name
     @lgsl = lgsl
     @country_name = country_name
     @local_authority_homepage = local_authority_homepage
-
-    @unavailable_content = LocalTransactionService.config.dig(
-      lgsl, "unavailable_in", @country_name
-    ) || {}
+    @devolved_administration_availability = devolved_administration_availability
   end
 
-  def body
-    @unavailable_content["body"]
+  def unavailable?
+    find_devolved_administration_availability[country_name] == "unavailable"
   end
 
-  def button_text
-    @unavailable_content["button_text"] || "Find other services"
+  def devolved_administration?
+    find_devolved_administration_availability[country_name] == "devolved_administration_service"
```

Comment:
Similar point here that we could moved some of the logic here into the presenter.